scripts/fabtests/ccm_runfabtests.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+#
+# Copyright (c) 2016 Cray Inc.  All rights reserved.
+# Copyright (c) 2016 Los Alamos National Security, LLC. All rights reserved.
+#
+# This software is available to you under a choice of one of two
+# licenses.  You may choose to be licensed under the terms of the GNU
+# General Public License (GPL) Version 2, available from the file
+# COPYING in the main directory of this source tree, or the
+# BSD license below:
+#
+#     Redistribution and use in source and binary forms, with or
+#     without modification, are permitted provided that the following
+#     conditions are met:
+#
+#      - Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      - Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials
+#        provided with the distribution.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AWV
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+debug=0
+load_ccm_module="module load ccm"
+ccm_login="ccmlogin"
+ccm_login_script="${HOME}/ccm_login_fabtests_script.$$"
+fabtests_directory="`pwd`"
+fabtests_cmd="./runfabtests.sh -p "
+provider="gni"
+test_suite=""
+use_parameters=0
+verbose=""
+
+usage() {
+    ec=$1
+    echo "cray_ccm_runfabtests.sh [-d <directory>] [-D] [-p <provider>] [-t <test_suite>] [-u] [-v] [-vv] [-vvv]"
+    echo "   -d libfabric tests location"
+    echo "   -D print debug output"
+    echo "   -p libfabric provider to use"
```

Comment:
This flag above is fine, but I don't think you need that `if [[ "${provider}" == "gni" ]]` clause below.